### PR TITLE
entity/relationship testworld tests

### PR DESCRIPTION
**Pull request by manuelpolzhofer**
- entity testworld tests
- default p2p request for entity relationship service

### CODE REVIEW

**Comment by manuelpolzhofer** [comment]
@charleenfei I opened the PR to see if all test pass

File: documents/entity/service.go [hunk 1]
```diff
@@ -210,13 +216,87 @@ func (s service) DeriveEntityResponse(model documents.Model) (*cliententitypb.En
...
 	}, nil
+}
+
+// DeriveEntityRelationshipData returns the relationship data from an entity relationship model
+func (s service) DeriveEntityRelationshipData(model documents.Model) (*cliententitypb.RelationshipData, error) {
+	return s.erService.DeriveEntityRelationshipData(model)
+}
+
+// DeriveRelationshipsListResponse returns a relationships list response from the array of relationship models
+func (s service) DeriveRelationshipsListResponse(entity documents.Model, models []documents.Model) (*cliententitypb.RelationshipResponse, error) {
```
Comment (mikiquantum) [inline documents/entity/service.go]: we do not need this right?

Comment (charleenfei) [reply documents/entity/service.go]: I've removed it in my PR, requested your review on Manuel's by mistake :)

**Comment by manuelpolzhofer** [comment]
I am going to close this one. Purpose was just testing

**Pull request closed without merging**